cibyl/sources/jenkins.py

```diff
@@ -191,6 +191,7 @@ def generate_query_url():
 
         return json.loads(response.text)
 
+    @speed_index({'jobs': 2, 'job_url': 2, 'builds': 1})
```

Comment:
If I understood correctly, builds score should not be here, but on the get_builds method

Reply:
Yes, can I please ask you make some changes to this PR based on your review and others' comments? I won't be near the computer anytime soon

Reply:
Sure, no problem